ros_ws/src/navigation_stack/car_cartographer/launch/cartographer_offline.launch

```diff
@@ -0,0 +1,46 @@
+<!--
+  Copyright 2016 The Cartographer Authors
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<!-- Modified by the F1/10 Autonomous Racing Project Group -->
+
+<launch>
+  <arg name="ros_version" default="kinetic"/>
+
+  <param name="/use_sim_time" value="true" />
```

Comment:
Is this used anywhere?

Reply:
Yes, it is internally used by cartographer. Note that the difference between `<arg>` and `<param>` is that an `<arg>` variable is confined to the launch file it is declared in while a `<param>` variable is pushed onto the parameter server and can be used by any ros node.